for some reason this configure test fails to detect egl x11 support.
we obviously have it, and the configure test works manually...
so just force it on
--
diff --git a/src/gui/configure.json b/src/gui/configure.json
index 1f08795c..bc120324 100644
--- a/src/gui/configure.json
+++ b/src/gui/configure.json
@@ -838,12 +838,7 @@
                 ],
                 "include": [ "EGL/egl.h", "X11/Xlib.h" ],
                 "main": [
-                    "Display *dpy = EGL_DEFAULT_DISPLAY;",
-                    "EGLNativeDisplayType egldpy = XOpenDisplay(\"\");",
-                    "dpy = egldpy;",
-                    "EGLNativeWindowType w = XCreateWindow(dpy, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0);",
-                    "XDestroyWindow(dpy, w);",
-                    "XCloseDisplay(dpy);"
+                    "return 0;"
                 ]
             },
             "use": "egl xlib"
